Riot/Modules/Spaces/SpaceMenu/SpaceMenuViewModel.swift

```diff
@@ -25,11 +25,12 @@ class SpaceMenuViewModel: SpaceMenuViewModelType {
     weak var viewDelegate: SpaceMenuViewModelViewDelegate?
 
     private let spaceMenuItems: [SpaceMenuListItemViewData] = [
+        SpaceMenuListItemViewData(action: .invite, style: .normal, title: VectorL10n.spacesInvitePeople, icon: Asset.Images.spaceInviteUser.image, value: nil),
         SpaceMenuListItemViewData(action: .exploreSpaceMembers, style: .normal, title: VectorL10n.roomDetailsPeople, icon: Asset.Images.spaceMenuMembers.image, value: nil),
+        SpaceMenuListItemViewData(action: .settings, style: .normal, title: VectorL10n.sideMenuActionSettings, icon: Asset.Images.sideMenuActionIconSettings.image, value: nil),
         SpaceMenuListItemViewData(action: .exploreSpaceRooms, style: .normal, title: VectorL10n.spacesExploreRooms, icon: Asset.Images.spaceMenuRooms.image, value: nil),
```

Comment:
shouldn't this be displayed over exploreSpaceMembers ? (according to #5810 initial message)

Reply:
You're completely right! Thanks for double checking. It should be the right order now :)